src/mbgl/tile/raster_dem_tile.hpp

```diff
@@ -11,6 +11,53 @@ class Tileset;
 class TileParameters;
 class HillshadeBucket;
 
+enum class DEMTileNeighbors : unsigned char {
+  // 0b00000000
+  Empty = 0 << 1,
+
+  // the order of the neighbors in this enum (or rather the order of their flipped bits
+  // must stay the same as
```

Comment:
Incomplete comment?

Reply:
derp – yeah I think I decided to use a lambda while writing this